sections/futures/PositionCard/ClosePositionModal.tsx

```diff
@@ -15,13 +15,11 @@ import GasPriceSelect from 'sections/shared/components/GasPriceSelect';
 import { getFuturesMarketContract } from 'queries/futures/utils';
 import Connector from 'containers/Connector';
 import Button from 'components/Button';
-import { getExchangeRatesForCurrencies, synthToAsset } from 'utils/currencies';
+import { newGetExchangeRatesForCurrencies, synthToAsset } from 'utils/currencies';
```

Comment:
Will we eventually be able to rename this, and deprecate the old one? I'm assuming the old one is still in use on old components/sections

Reply:
Yeah, exactly @Tburm